subprojects/errorprone/src/test/java/org/mockito/errorprone/bugpatterns/MockitoMockedStaticTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2017 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.errorprone.bugpatterns;
+
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class MockitoMockedStaticTest {
```

Comment:
I think you missed the case where you have a `@Mock private MockedStatic<Object>` in the original issue description. You can use `// BUG: diagnostic contains:` to verify that the compilation would fail. Example: https://github.com/google/error-prone/blob/18d5cdf10ec1cc798a588d5c48a9e02a8888c6a1/core/src/test/java/com/google/errorprone/bugpatterns/ArrayAsKeyOfSetOrMapTest.java#L60